docs/02-Install.md

```diff
@@ -93,13 +93,17 @@ Typically, images insertion into LORIS is performed via the following steps:
      study-defined protocol:
      
     a. If a scan matches one of the protocol defined in the `mri_protocol`
-         table, then the MINC image will be stored into the `files` tables.
-         This inserted image is then accessible via the **Imaging Browser**
-         module and can be displayed in 3D using **BrainBrowser**.
+         table and passes the optional additional file checks present in the 
+         `mri_protocol_checks` table, then the MINC image will be stored into 
+         the `files` tables. This inserted image is then accessible via the 
+         **Imaging Browser** module and can be displayed in 3D using 
+         **BrainBrowser**.
 
     b. If a scan does not match any of the protocol defined in the
-         `mri_protocol` table, then the MINC image of this scan will be stored
-         into the `mri_protocol_violated_scans` table. This violated image is
+         `mri_protocol` table or is excluded by the optional additional file 
+         checks present in the `mri_protocol_checks` table, then the MINC 
+         image of this scan will be stored into the 
+         `mri_protocol_violated_scans` table. This violated image is
```

Comment:
I believe that if the protocol_checks return the excluded outcome, the logging is not done in mri_violated_scans but rather in mri_violations_log.